src/main/controllers/helpers/CitizenHubHelper.ts

```diff
@@ -81,3 +81,17 @@ export const userCaseContainsGeneralCorrespondence = (notifications: SendNotific
     notifications.some(it => it.value.sendNotificationSubject.includes(NotificationSubjects.GENERAL_CORRESPONDENCE))
   );
 };
+
+export const checkIfRespondentIsSystemUser = (userCase: CaseWithId): boolean => {
+  const repCollection = userCase.representatives;
+  const respondentCollection = userCase.respondents;
+
+  return (
+    respondentCollection !== undefined &&
+    repCollection !== undefined &&
+    respondentCollection.every(res => {
+      return repCollection.some(rep => res.ccdId === rep.respondentId);
```

Comment:
This can be returned immediately, so `respondentCollection.every(res => repCollection.some(rep => res.ccdId === rep.respondentId)`

Reply:
ignoring this one as discussed

Reply:
Unit test this and remove the return, you should be able to do this